### PR TITLE
Wave 6 character Ids

### PR DESCRIPTION
**Pull request by mrbaboo**

### CODE REVIEW

File: data/charactermap.json [hunk 1]
```diff
@@ -1,48 +1,56 @@
...
+{ "id":38,	"name":"Shaggy",				"world":"Scooby-Doo" },
+{ "id":39,	"name":"Stay Puft",				"world":"Ghostbusters" },
+{ "id":40,	"name":"Superman",				"world":"DC Comics" },
+{ "id":41,	"name":"Unikitty",				"world":"The LEGO Movie" },
+{ "id":42,	"name":"Wicked Witch",			"world":"Wizard of OZ" },
+{ "id":43,	"name":"Superwoman",			"world":"DC Comics" },
+{ "id":44,	"name":"Zane",		 			"world":"Ninjago" },
+{ "id":45,	"name":"Green Arrow",			"world":"DC Comics" },
+{ "id":46,	"name":"Supergirl",				"world":"DC Comics" },
+{ "id":47,	"name":"Abby Yates",			"world":"New Ghostbusters" },
```
Comment (mrbaboo) [inline data/charactermap.json]: Everything above this line was just a change to align with the new position of `"world"`

Comment (mrbaboo) [inline data/charactermap.json]: Everything below are the new IDs. Note that ID 54 is currently unassigned

**Comment by matthewbullweb** [comment]
So old ids remain the same and they just continue new one on?

**Comment by ags131** [comment]
@mrbaboo Thanks!
@matthewbullweb Correct, old IDs should never change.

**Pull request merged**